routes/tradeHistory.js

```diff
@@ -0,0 +1,17 @@
+import {Router} from "express";
+import { postHistory } from "../controllers/tradeHistory";
+const router = Router()
+
+
+/**
+ * @swagger
+ * /tradeHistory:
+ *   post:
+ *     summary: help check if the customer make payment
+ *     responses:
+ *       '200':
+ *         description: A successful response
+ */
+ router.post('/tradeHistory', postHistory)
```

Comment:
probably should be called **transactionHistory**?

Reply:
I have no idea, because TapPay, which offer line pay functionality, use this term in its document